esdoc.json

```diff
@@ -38,6 +38,10 @@
       "./doc/tutorial/basics.md",
       "./doc/tutorial/applications.md",
       "./doc/external-references.md"
+    ],
+    "usage": [
+      "./doc/writing-marble-tests.md",
+      "./doc/operator-creation.md"
```

Comment:
How would this `usage` section be rendered on the website? Can you upload the generated site to some easy link to surge.sh or now.sh or netlify? It would help us know how the site would look like after these changes.